src/lib/common/versionInfo.h

```diff
@@ -0,0 +1,42 @@
+#ifndef SRC_LIB_COMMON_VERSIONINFO_H_
+#define SRC_LIB_COMMON_VERSIONINFO_H_
+
+/*
+*
+* Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
+*
+* This file is part of Orion Context Broker.
+*
+* Orion Context Broker is free software: you can redistribute it and/or
+* modify it under the terms of the GNU Affero General Public License as
+* published by the Free Software Foundation, either version 3 of the
+* License, or (at your option) any later version.
+*
+* Orion Context Broker is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+* General Public License for more details.
+*
+* You should have received a copy of the GNU Affero General Public License
+* along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+*
+* For those usages not covered by this license please contact with
+* iot_support at tid dot es
+*
+* Author: Burak Karaboga - ATOS Resarch and Innovation
+*/
+
+/* ****************************************************************************
+*
+* Release Date - The date of this version's release in YYYY-MM-DD format but other formats are also allowed.
+*/
+#define RELEASE_DATE  "Mon Sep 11 19:22:01 CEST 2017"

```

Comment:
This kind of "dynamic" information is not directly included in source files. Instead of that, the scripts/build/compileInfo.sh is used (note that that script is invoked by `makefile` in order to generate the proper .h).

In that sense, compilseInfo.sh should be modified to:

* Include the generation of a new define for the release date (just after `echo '#define COMPILED_IN      "'${host}'"' <...>`)
* Generate the date in a similar way currently compiled date is done. In fact, probably the same `${date}` could be reused.

Reply:
My understanding was that the release_date points to the date of the version release on Github. Please correct me if I am wrong, if we use the same date with the compile date, every running instance of Orion built from the sources would produce different outputs for the release date even though they might be on the same version (e.g 1.8.0).

Just to be clear for everybody, can you please clarify what is intended to be displayed by the release_date?

Reply:
In that case, release date could be an optinal parameter of the reselase.sh script, defaulting to system date in the moment of running that script (that is, `${date}`) if no release date is provided in its CLI.

Reply:
> Just to be clear for everybody, can you please clarify what is intended to be displayed by the release_date?

According to https://docs.google.com/document/d/1hJgawz4ibu1OBA_OlqVfZY7fzIHQ0c4vd5b-T7r2nHM/edit

> Release_date: I[t] shows the date of the release

Maybe in some other GEri makes sense to distinguish between compilation date and release date but, in the case Orion, each time a release is done a compilation is also done, synchronous at the same time. Thus, it make sense from my point of view to have "release_date" to be equal to "compiled_time" in the case of Orion (although the alternative I have described in the previour comment, an optional parameter of releases.sh scripot, defaulting to `${date}` is also fine).

Reply:
If this is the case, in order not to touch more files than intended, I will go with using the compiled_time value for the release_date. Thanks for your comments.

Reply:
Fixed in e88551b